### PR TITLE
New layout structure documentation

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
Documentation to describe the new layout structure

### CODE REVIEW

File: docs/incidents/incident-customize-incident-layout.md [hunk 1]
```diff
@@ -76,3 +76,1470 @@ To change the information that appears in dynamic sections:
 
 You can add new sections or fields to the layout by dragging and dropping them from the **Library** on the left into the layout. For example, insert a new field that you created into a new, or already existing, section.
 
+## Layout file structure
+
+Layout structure differs before and after Cortex XSOAR version 6.0.
+
+#### Up to Cortex XSOAR version 6.0
+
+Each layout kind should be represented in one file, as exported from Cortex XSOAR, with addition of the field `toVersion: 5.9.9`.
```
Comment (anara123) [inline docs/incidents/incident-customize-incident-layout.md:85]:
```suggestion
Each layout kind should be represented in one file, as exported from Cortex XSOAR, with addition of the field `"toVersion": "5.9.9"`.
```

File: docs/incidents/incident-customize-incident-layout.md [hunk 2]
```diff
@@ -76,3 +76,1470 @@ To change the information that appears in dynamic sections:
...
+    "quickView": object,            // Optional layout type
+    "mobile": object,               // Optional layout type
+    "indicatorsDetails": object,    // Optional layout type
+    "indicatorsQuickView": object   // Optional layout type
+ }
+ ```
+ 
+ For example:
+ ```json
+ {
```
Comment (anara123) [inline docs/incidents/incident-customize-incident-layout.md:116]: maybe instead of adding this big json here, add a link to real layout from content

Comment (Itay4) [reply docs/incidents/incident-customize-incident-layout.md:116]: will do once the content pr is merged

File: docs/incidents/incident-customize-incident-layout.md [hunk 3]
```diff
@@ -76,3 +76,1470 @@ To change the information that appears in dynamic sections:
...
+#### Up to Cortex XSOAR version 6.0
+
+Each layout kind should be represented in one file, as exported from Cortex XSOAR, with addition of the field `"toVersion": "5.9.9"`.
+
+The file should be named `layout-<KIND>-<TYPE>.json`, e.g. `layout-details-Phishing.json`
+
+#### Cortex XSOAR version 6.0 and above
+
+Cortex XSOAR version 6.0 introduces better layout management. As part of it, all the kinds and types (i.e. detailsV2, edit, etc...) of layouts are consolidated in one file, as described below.
+
```
Comment (glicht) [inline docs/incidents/incident-customize-incident-layout.md:92]: Is there need to do anything after exporting the file from Server? Let's add a one liner explaining the export process.

Comment (Itay4) [reply docs/incidents/incident-customize-incident-layout.md:92]: basically just needs to add the `fromVersion` as the file already includes all required fields (and more)

we will need to add support to the `format` cmd to add the `fromVersion` and remove garbage fields

**Comment by Itay4** [comment]
@glicht added a note about the incident/indicator type association

**Comment by Itay4** [comment]
@glicht @anara123 added link to phishing layout instead of json example

**Pull request merged**